pkg/guardian/guardian_test.go

```diff
@@ -23,13 +23,22 @@ func newAcceptanceGuardianServer(t *testing.T, logger logrus.FieldLogger) (*Serv
 
 	stop := make(chan struct{})
 	redis := redis.NewClient(&redis.Options{Addr: mr.Addr()})
+	if res := redis.Ping(); res.Err() != nil {
```

Comment:
I assume this was done out of necessity? Super curious as to why. IIRC, mr is a mock redis that runs in process so it should never really go down.

Reply:
IIRC I was getting connection refused errors prior to this change. 🤷‍♂